cypress/e2e/ui-tests/test-deploy-app.cy.js

```diff
@@ -2,42 +2,49 @@ describe("Test deploying app", () => {
 
     // Tests performed as an authenticated user that
     // creates and deletes objects.
-    // user: e2e_tests_contributor_tester
+    // user: e2e_tests_deploy_app_user
 
     let users
 
     before({ defaultCommandTimeout: 100000 }, () => {
-        // seed the db with: contributor user, a blank project
-        cy.log("Seeding the db for the contributor tests. Running db-seed-contributor.sh");
-        cy.exec("./cypress/e2e/db-reset.sh")
-        cy.wait(60000)
+        // do db reset if needed
+        if (Cypress.env('do_reset_db') === true) {
+            cy.log("Resetting db state. Running db-reset.sh");
+            cy.exec("./cypress/e2e/db-reset.sh");
+            cy.wait(60000);
```

Comment:
I would like to see the cypress wait timeout as a configurable setting so it can be tuned to the machine and environment running the tests. This 60000 ms wait is used in many places.

Reply:
good idea! I can make it into a variable in `cypress.config.js`.

Reply:
Done now.